po/nl.po

```diff
@@ -93,11 +93,13 @@ msgid "<loglevel>"
 msgstr "<niveau>"
 
 #: newsboat.cpp:81
-#, fuzzy
 msgid ""
 "write a log with a certain log level (valid values: 1 to 6, for user error, "
 "critical, error, warning, info, and debug respectively)"
-msgstr "log niveaus 1..<niveau> wegschrijven (geldige waarden: 1 tot 6)"
+msgstr ""
+"log niveaus 1..<niveau> wegschrijven (geldige waarden: 1 tot 6, "
```

Comment:
The "1..<niveau>" looks a bit odd to me. But I still haven't learned Dutch. ;-)

Reply:
I think that was a remainder of a translation of older source material.
I've tried to make it slightly more regular.

By the way, nice to see all of those quick language reviews by you!